### ISSUE TITLE
Mexico Competitions Displaying Incorrect Time on Calendar

### ISSUE THREAD

**Issue by dunkOnIT** [thewca/worldcubeassociation.org#7596]
Email thread: "Issue with WCA competition Site"

> We have the theory this happened because of Daylight Saving Time.
> 
> Usually, we in Mexico set forward one-hour clocks in summer but last year the president changed the law and this Daylight Saving time was canceled. You can find some news like this https://elpais.com/mexico/2022-10-26/mexico-elimina-el-horario-de-verano-tras-26-anos-de-implementarlo.html#:~:text=Este%20domingo%2030%20de%20octubre,tengan%20que%20atrasar%20su%20reloj.
> 
> I noticed this issue affected all the competitions after April in Mexico. So we suppose this is the issue. How can we actualize this information and eliminate Daylight Saving Time?
> 
> The important competition affected are:
> https://www.worldcubeassociation.org/competitions/IZcube2023
> https://www.worldcubeassociation.org/competitions/CalmecacTlaxcala2023

**Comment by FinnIckler** [comment]
Did some digging by creating a Mexico Competition on Staging.
This is the js that ruby generates for the schedule data with a competition using the Mexico City timezone:
```
      window.wca.registerVenueData("4479", {
        events: [{...
        "start":"2023-06-14T11:00:00.000-06:00","end":"2023-06-14T12:00:00.000-06:00}],
        minTime: "13:00:00",
        maxTime: "14:59:59",
      });
```
Which is the correct timezone for mexico city, but on the competition edit schedule it is shown wrong.
So my theory is that it's a js issue

**Comment by dunkOnIT** [comment]
Thanks for looking into it! For clarity, does it seem to be an issue baked into the JS language, rather than our implementation of it?

**Comment by FinnIckler** [comment]
No, I mean it's an issue with the library we are using to build the schedule which is js based

**Comment by dunkOnIT** [comment]
Do we know which JS library it is? I'll go log an issue on the relevant repo once we know which one

**Comment by FinnIckler** [comment]
We are using fullcalendar.js, but I can't find any reference to where it gets the timezone in https://github.com/thewca/worldcubeassociation.org/blob/0c0a3f44f32b4461f9d40e01fd3000b30fedfd66/WcaOnRails/app/webpacker/lib/helpers/fullcalendar.js or https://github.com/thewca/worldcubeassociation.org/blob/0c0a3f44f32b4461f9d40e01fd3000b30fedfd66/WcaOnRails/app/webpacker/lib/fullcalendar.js
So it might be something else that gives it to fullcalendar...
